src/tscore/ink_inet.cc

```diff
@@ -463,6 +463,27 @@ IpAddr::toString(char *dest, size_t len) const
   return dest;
 }
 
+sockaddr *
+IpAddr::toSockAddr(sockaddr *dest) const
+{
+  if (AF_INET == _family) {
```

Comment:
maybe use a **switch(_family)**?

Reply:
It's only two values and those values are not compact, not sure a `switch` makes it any better.